sdk/storage/azblob/service/client_test.go

```diff
@@ -680,8 +691,20 @@ func (s *ServiceUnrecordedTestsSuite) TestContainerRestore() {
 
 			if *cont.Deleted && *cont.Name == containerName {
 				contRestored = true
-				_, err = svcClient.RestoreContainer(context.Background(), containerName, *cont.Version, nil)
-				_require.Nil(err)
+				const maxTries = 5
+				for tries := 0; tries < maxTries; tries++ {
+					_, err = svcClient.RestoreContainer(context.Background(), containerName, *cont.Version, nil)
```

Comment:
This consistently fails on the first run, and the retries don't help.  Rerunning the test passes.

Reply:
Fixed, see code comments for details.